liquidprompt

```diff
@@ -1840,6 +1845,8 @@ _lp_error_meaning_color() {
 
 # shellcheck disable=SC2034
 _lp_multiplexer() {
+    (( LP_ENABLE_SCREEN_TITLE || LP_ENABLE_TMUX_TITLE_PANES || LP_ENABLE_MULTIPLEXER )) || return 2
```

Comment:
This does work, but I'm not a fan of how it works. This means any theme that calls `_lp_multiplexer()` has to also check the value of `LP_ENABLE_MULTIPLEXER`. That would need good documentation, and users would still probably get it wrong.

Why not add an argument to this function, where `liquidprompt` can set it, where it checks the screen vars instead?

```suggestion
    if [[ ${1-} == "--internal" ]]; then
        (( LP_ENABLE_SCREEN_TITLE || LP_ENABLE_TMUX_TITLE_PANES)) || return 2
    else
        (( LP_ENABLE_MULTIPLEXER )) || return 2
    fi
```

Then where we call this function to set the right screen escape string, we can pass this `--internal` (I am not attached to that name) argument.